vspace/src/main/java/edu/asu/diging/vspace/web/staff/DeleteModuleController.java

```diff
@@ -0,0 +1,37 @@
+package edu.asu.diging.vspace.web.staff;
+
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import edu.asu.diging.vspace.core.services.IModuleManager;
+
+@Controller
+public class DeleteModuleController {
+    
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+    
+    @Autowired
+    private IModuleManager moduleManager;
+
+    @RequestMapping(value="/staff/module/{moduleId}/delete", method=RequestMethod.DELETE)
+    public ResponseEntity<String> deleteModule(@PathVariable("moduleId") String moduleId, Model model) {
+        try {
+            moduleManager.deleteModule(moduleId);
+        }
+        catch (Exception exception) {
```

Comment:
`deleteModule` doesn't seem to be throwing any exceptions. What are you handling here?

Reply:
Added illegalstateexception in case module is not found